packages/preact/index.js

```diff
@@ -53,4 +54,12 @@ module.exports = (neutrino, opts = {}) => {
         .set('react-dom', 'preact-compat')
         .set('create-react-class', 'preact-compat/lib/create-react-class')
         .set('react-addons-css-transition-group', 'preact-css-transition-group');
+
+  neutrino.config.when(neutrino.config.module.rules.has('lint'), () => {
```

Comment:
Was this change related to the rest of the PR? I couldn't see anything explaining it in the commit message or review comments?